src/utils/records-decorator.js

```diff
@@ -3,7 +3,6 @@ const _ = require('lodash');
 function decorateForSearch(records, fields, searchValue) {
   const matchFields = {};
   records.forEach((record, index) => {
-    record = record.toObject();
```

Comment:
Are you confident enough for this change?

Reply:
If I leave this line it doesn't works but when I remove it it works. I think it's because the object is not exactly the same from find and aggregate